src/index.html

```diff
@@ -11,7 +11,154 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    onsubmit="onSubmit()"
+    >
```

Comment:
Додай ще метод до форми
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      onsubmit="onSubmit()"
    >
```